docs/roadmap.md

```diff
@@ -124,11 +126,7 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 ### DevOps:
 
 - [x] Upgrade Jenkins Version to 2.249.1. [#2618](https://github.com/kubesphere/kubesphere/issues/2618)
-- [x] Using Jenkins distribution solution to deploy Jenkins, [#2182](https://github.com/kubesphere/kubesphere/issues/2182)
-- [x] Using human-readable error message for pipeline cron text , [#2919](https://github.com/kubesphere/kubesphere/issues/2919)
-- [ ] Using human-readable error message for S2I, [#140](https://github.com/kubesphere/s2ioperator/issues/140)
-
-- [ ] https://github.com/kubesphere/tower/issues/31)
```

Comment:
hi @yuswift , this record should be here. I don't know where this record should be.

see also https://github.com/kubesphere/kubesphere/pull/3412/files#diff-6a17656d764b8e1930354f341cf7f47960b63845c3046edcd8bbf282aaa2f464R130